core/src/main/java/com/linecorp/armeria/common/metric/AbstractMetricCollectingBuilder.java

```diff
@@ -83,4 +87,21 @@ public AbstractMetricCollectingBuilder successFunction(
         this.successFunction = requireNonNull(successFunction, "successFunction");
         return this;
     }
+
+    /**
+     * Returns the {@code distributionStatisticConfig}.
+     */
+    protected final DistributionStatisticConfig distributionStatisticConfig() {
+        return distributionStatisticConfig;
+    }
+
+    /**
+     * Defines a custom {@link DistributionStatisticConfig} to use for the distribution config.
+     */
+    public AbstractMetricCollectingBuilder distributionStatisticConfig(
```

Comment:
Shouldn't we also override this method in child classes?

i.e. the following won't compile:
```java
MetricCollectingClient.builder(MeterIdPrefixFunction.ofDefault("armeria"))
                      .distributionStatisticConfig(DistributionStatisticConfig.DEFAULT)
                      .newDecorator()
```

Reply:
> Shouldn't our builder test (com.linecorp.armeria.OverriddenBuilderMethodsReturnTypeTest) catch this issue automatically?

The builder isn't final 😅 

https://github.com/line/armeria/blob/5df7f1628daec5fbf7fc049743f39193cc7f33de/it/builders/src/test/java/com/linecorp/armeria/OverriddenBuilderMethodsReturnTypeTest.java#L51